### PR TITLE
Fixed non-arsenal equipment and added Winter Faction

### PR DESCRIPTION
**Pull request by tmthwoods**
Fixed non-arsenal equipment

### CODE REVIEW

**Comment by metformin21** [comment]
faction files are @Amateur-God's repertoire so I'm not too familiar, I'm willing to merge this after skimming it though if he isn't going to look

**Comment by Amateur-God** [comment]
Will have a look when I'm at my PC maybe tomorrow if that's okay @tmthwoods

File: addons/UK_2BNB/infantry/Desert.hpp
Comment (Amateur-God) [inline addons/UK_2BNB/infantry/Desert.hpp:1]: @tmthwoods are we no longer using the 3CB smokeshells?

Comment (Amateur-God) [reply addons/UK_2BNB/infantry/Desert.hpp:1]: or any 3CB GL rounds?

File: addons/UK_2BNB/weapons/index.hpp
Comment (Amateur-God) [inline addons/UK_2BNB/weapons/index.hpp:1]: did i put the ; by mistake when i wrote this originally?

**Review by Amateur-God — CHANGES_REQUESTED** [review]
Didnt update index,hpp to include winter

File: addons/UK_2BNB/infantry/Desert.hpp [hunk 1]
```diff
@@ -47,8 +47,8 @@
...
-    linkedItems[]={"ItemMap","ItemCompass","ItemWatch","TFAR_anprc152","ItemGPS","UK3CB_BAF_Soflam_Laserdesignator","UK3CB_BAF_H_Mk6_DDPM_C","UK3CB_BAF_V_Osprey_DDPM3","UK3CB_BAF_G_Tactical_Black"};
-    respawnLinkedItems[]={"ItemMap","ItemCompass","ItemWatch","TFAR_anprc152","ItemGPS","UK3CB_BAF_Soflam_Laserdesignator","UK3CB_BAF_H_Mk6_DDPM_C","UK3CB_BAF_V_Osprey_DDPM3","UK3CB_BAF_G_Tactical_Black"};
```
Comment (Amateur-God) [inline addons/UK_2BNB/infantry/Desert.hpp:51]: Deleted the " at the end of DDPM3 on respawn linked items breaking the quotes

Comment (Amateur-God) [reply addons/UK_2BNB/infantry/Desert.hpp:51]: needs replacing so arma can read it properly

**Review by Amateur-God — CHANGES_REQUESTED** [review]
add " back into respawnLinkedItems

**Review by Amateur-God — APPROVED** [review]

**Pull request merged**